drivers/dai/intel/ssp/ssp.c

```diff
@@ -1009,43 +991,38 @@ static void ssp_empty_rx_fifo_on_stop(struct dai_intel_ssp *dp)
 	uint32_t i, sssr, ssmidycs;
 	uint32_t entries[2];
 
-	if (ssp_is_acquired(dp->ssp_plat_data)) {
-		return;
-	}
-
```

Comment:
It would be good to have some link to the commit this is fixing, i.e. a2386efbce1866613f7c714b958a159364dc5f37 ("drivers: ssp: update SSP driver to support Intel ACE30 PTL").

We are in freeze for 3.7.0, so only bugfixes can go in, and with the current description, I don't think this can be merged this week. 

But, but, what are the minimal changes to fix non-PTL supportt. Is this this line L1012... or:

Reply:
I'm afraid that the minimal changes required to fix non-PTL support are in this PR. After adding https://github.com/zephyrproject-rtos/zephyr/commit/a2386efbce1866613f7c714b958a159364dc5f37 ("drivers: ssp: update SSP driver to support Intel ACE30 PTL"), the issue with the unbalanced acquire/release counter was uncovered for LNL (pause-resume repeat scenario). Before merging the mentioned commit, the problem existed, but the tests did not catch it. I will try to find an easier way to fix the pre-PTL platform.